README.md

```diff
@@ -1,30 +1,156 @@
+# CanDIGv2 Playwright Runbook
 
-# Installing Playwright
+This repository provides a Playwright-based framework for automated UI and API performance testing of CanDIG systems. It includes scripts to run tests on local, development, or production server (with limited access) to make sure both frontend and backend meet expectations.
+
+## Installing Playwright
+
+[Playwright](https://playwright.dev/) requires [Node.js](https://nodejs.org/en/download/package-manager).
+
+Clone the repository and execute the following commands:
 
 ```bash
+cd playwright-runbook
 npm install
+npx playwright install
 ```
 
-# Set .env
+## Configuring Environment Variables
+
+Rename `.env.sample` to `.env`, and update the environment values. Credentials can be found in the `CanDIGv2/tmp/keycloak` folder.
+
+For local testing:
+
+- Set the URL to `localhost` or `candig.docker.internal`.
+
+For dev testing:
+
+- Set the URL to your dev server (e.g., `candig-dev.hpc4health.local`)
+
+For prod testing:
+
+- Set the URL to your production server (e.g., `candig.uhnresearch.ca`).
 
-Rename `.env.sample` to `.env` and modify the values
+Note: The `QUERY_SERVICE_TOKEN` can be ignored unless you want to run katsu explorer api.
 
-# Running the Test
+## Running Tests
+
+Once the URL is set and the target server (local/dev/prod) is running, you can run all tests by using:
```

Comment:
I think if this is not the way you expect people to run the tests, it should go after the ways below, rather than first

Reply:
should I take it out?